frontend/js/src/settings/import/ImportListens.tsx

```diff
@@ -148,6 +160,7 @@ export default function ImportListens() {
 
   const [loading, setLoading] = React.useState(false);
   const [imports, setImports] = React.useState<Array<Import>>([]);
+  const [fileSelected, setFileSelected] = React.useState<File | null>(null);
```

Comment:
I am doubting why you need to store the entire file in memory instead of a simple boolean, if this is only used for disabling the submit button.